### PR TITLE
OmegaScans : From MangaStream to Custom API

### LINKED ISSUE #5633: [OmegaScans] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Did you [try Nightly Build](https://github.com/manga-download/hakuneko/releases) ?**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
When I want to refresh the list of manga or chapters of a manga, I get a 404 error.
The site seems to have been updated and no longer use WordPress Mangastream.

**To Reproduce**
1. Name of connector : OmegaScans
2. Name of manga (if connector update is working) : not working
3. Name of chapter (if manga update is working) : not working
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12) : 

GET https://omegascans.org/manga/list-mode 404
fetchDOM	@	Connector.mjs:434
_getMangas	@	WordPressMangastream.mjs:25
_getMangaList	@	Connector.mjs:576
(anonymous)	@	Connector.mjs:99
Promise.then (async)		
updateMangas	@	Connector.mjs:98
onUpdateMangaListClick	@	mangas.html:233
handler	@	template-stamp.html:96

**HakuNeko (please complete the following information):**
 - Version [master@deed91]

### PR DESCRIPTION
**Pull request by MikeZeDev**
Similar to ReaperScans family.

Fixes https://github.com/manga-download/hakuneko/issues/5633

### CODE REVIEW

**Comment by xavimaa** [comment]
First thank you for your work !
I tried your commit yesterday and it worked but today I got a 404 error on the getChapterList function

**Comment by MikeZeDev** [comment]
> First thank you for your work ! I tried your commit yesterday and it worked but today I got a 404 error on the getChapterList function

Yeah. nextInstance changed. I updated the code so we use the actual one instead of an hard-coded one.

File: src/web/mjs/connectors/OmegaScans.mjs [hunk 1]
```diff
@@ -1,13 +1,85 @@
...
-        this.path = '/manga/list-mode/';
+        this.api = 'https://api.omegascans.org';
+        this.nextInstance = 'Uzf9L765by7rm6wVbv5Sb';
+    }
+
+    async _initializeConnector() {
+        const uri = new URL(this.url);
+        const request = new Request(uri.href, this.requestOptions);
+        this.nextInstance = await Engine.Request.fetchUI(request, `__NEXT_DATA.buildId`);
+        console.log(`OmegaScans nextInstance is  ${this.nextInstance}`);
```
Comment (27136773) [inline src/web/mjs/connectors/OmegaScans.mjs]: Is this `log` left by intention for production, or just forgotten to remove after finishing development/investigation?

**Review by 27136773 — APPROVED** [review]

File: src/web/mjs/connectors/OmegaScans.mjs (hunk 1 above)
Comment (MikeZeDev) [reply src/web/mjs/connectors/OmegaScans.mjs]: Its intended but can be removed if you want

**Pull request merged**